lib/components/SLoginPage.vue

```diff
@@ -0,0 +1,207 @@
+<script setup lang="ts">
+import { computed } from 'vue'
+import SButton from './SButton.vue'
+import SLink from './SLink.vue'
+import SIconGbLogoWhite from './icon/SIconGbLogoWhite.vue'
+import SIconGoogle from './icon/SIconGoogle.vue'
+
+export interface CoverTitle {
+  text: string
+  link: string
+}
+
+export interface CoverPhotographer {
+  text: string
+  link: string
+}
+
+export interface Action {
+  type: 'google'
+  label?: string
```

Comment:
We might not need `label` prop to override login button text and just calculate it in `getActionLabel` function.
What do you think?

Reply:
Yeah I agree, we don't need to pass in the text I think. I'll update it along with other style fixes 👍